srcpkgs/ptunnel-ng/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'ptunnel-ng'
+pkgname=ptunnel-ng
+version=1.42
+revision=1
+build_style=configure
```

Comment:
```suggestion
build_style=gnu-configure
```

Reply:
and then you might be able to remove $configure_args